examples/scenarioDeployingSolarArrays.py

```diff
@@ -0,0 +1,724 @@
+    array2Element1PrescribedDataLog = array2RotProfilerList[0].spinningBodyOutMsg.recorder(dataRecRate)
+    array2Element2PrescribedDataLog = array2RotProfilerList[1].spinningBodyOutMsg.recorder(dataRecRate)
+    array2Element3PrescribedDataLog = array2RotProfilerList[2].spinningBodyOutMsg.recorder(dataRecRate)
+    array2Element4PrescribedDataLog = array2RotProfilerList[3].spinningBodyOutMsg.recorder(dataRecRate)
+    array2Element5PrescribedDataLog = array2RotProfilerList[4].spinningBodyOutMsg.recorder(dataRecRate)
+    array2Element6PrescribedDataLog = array2RotProfilerList[5].spinningBodyOutMsg.recorder(dataRecRate)
+    array2Element7PrescribedDataLog = array2RotProfilerList[6].spinningBodyOutMsg.recorder(dataRecRate)
+    array2Element8PrescribedDataLog = array2RotProfilerList[7].spinningBodyOutMsg.recorder(dataRecRate)
+    array2Element9PrescribedDataLog = array2RotProfilerList[8].spinningBodyOutMsg.recorder(dataRecRate)
+    array2Element10PrescribedDataLog = array2RotProfilerList[9].spinningBodyOutMsg.recorder(dataRecRate)
+    
+    scSim.AddModelToTask(fswTaskName, scStateData)
+    scSim.AddModelToTask(fswTaskName, array1Element1PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array1Element2PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array1Element3PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array1Element4PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array1Element5PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array1Element6PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array1Element7PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array1Element8PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array1Element9PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array1Element10PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array2Element1PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array2Element2PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array2Element3PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array2Element4PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array2Element5PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array2Element6PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array2Element7PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array2Element8PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array2Element9PrescribedDataLog)
+    scSim.AddModelToTask(fswTaskName, array2Element10PrescribedDataLog)
+
+    # Set up Vizard visualization
+    scBodyList = [scObject]
+    for i in range(num_elements):
+        scBodyList.append(["Array1Element" + str(i+1), array1ElementList[i].prescribedMotionConfigLogOutMsg])
+        scBodyList.append(["Array2Element" + str(i+1), array2ElementList[i].prescribedMotionConfigLogOutMsg])
+
+    viz = vizSupport.enableUnityVisualization(scSim, dynTaskName, scBodyList, saveFile=filename)
```

Comment:
Put line 401 on 3 lines with `saveFile=filename` on its own line that is commented out.   See `scenarioBasicOrbit.py`.  This way the use can quickly enable saving off the Vizard binary, but we don't create this file by default.

Reply:
Also, put all the vizInterface code after this check:
```if vizSupport.vizFound:```
This way, if BSK is built without vizInterface support this script will still execute.
The if statement should cover line 401 to 421

Reply:
Done!